### ISSUE TITLE
What is this?

### ISSUE THREAD

**Issue by dandv** [mls-fe/cage#3]

**Comment by sunnylost** [comment]
This was an private project and only dedicated for private use.
I'm really curious how did you find this?

**Comment by dandv** [comment]
If it's private then perhaps it should be in a private repo on GitHub?

I found it while searching GitHub.

**Comment by sunnylost** [comment]
yeah……what i'm really mean is "for most people it's useless, not for common use".

**Comment by dandv** [comment]
Sure. The problem is it shows up in search results instead of other projects that users are actually hoping to find.

**Comment by sunnylost** [comment]
Which keywords did you use?  I didn't expect this could confuse other people.
